textproc/utfcpp/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        nemtrif utfcpp 4.0.6 v
+revision            0
+categories          textproc
+license             Boost-1
+maintainers         nomaintainer
+
+description         UTF-8 with C++ in a Portable Way
+
+long_description    ${description}
+
+homepage            https://github.com/nemtrif/utfcpp
```

Comment:
`homepage` gets implicitly set by `github.setup`. You do not to include this line explicitly, and it can be dropped.